docs/sources/plugins/developing/plugin.json.md

```diff
@@ -12,17 +12,28 @@ weight = 8
 
-## Plugin JSON Schema
+## Plugin.json Schema
 
-| Property | Description |
-| ------------- |-------------|
-| id | unique name of the plugin - [conventions described in styleguide]({{< relref "code-styleguide.md" >}}) |
-| type | panel/datasource/app |
-| name | Human readable name of the plugin |
-| info.description | Description of plugin. Used for searching grafana.com plugins |
-| info.author | |
-| info.keywords | plugin keywords. Used for search on grafana net|
-| info.logos | link to project logos |
-| info.version | project version of this commit. Must be semver |
-| dependencies.grafanaVersion | Required grafana backend version for this plugin |
-| dependencies.plugins | required plugins for this plugin. |
+| Property                    | Description                                                                                                                   |
+| --------------------------- | ----------------------------------------------------------------------------------------------------------------------------- |
+| id                          | Unique name of the plugin. See [naming conventions described in styleguide]({{< relref "code-styleguide.md" >}})              |
+| type                        | One of `app`, `datasource`, `panel`                                                                                           |
+| name                        | Human-readable name of the plugin                                                                                             |
+| dependencies.grafanaVersion | Required Grafana version for this plugin                                                                                      |
+| dependencies.plugins        | An array of required plugins on which this plugin depends                                                                     |
+| info.author.name            | Author's name                                                                                                                 |
+| info.author.url             | Link to author's website                                                                                                      |
+| info.description            | Description of plugin. Used for search on grafana.com                                                                         |
+| info.keywords               | Array of plugin keywords. Used for search on grafana.com                                                                      |
+| info.links                  | An array of link objects to be displayed on this plugin's project page in the form `{name: 'foo', url: 'http://example.com'}` |
+| info.logos.small            | Link to small project logo                                                                                                    |
+| info.logos.large            | Link to large project logos                                                                                                   |
```

Comment:
logos should be logo in description area (same as line above).  This should also be an SVG file, and they can be the same.

Reply:
Done, and done